browser/ui/toolbar/brave_vpn_menu_model.cc

```diff
@@ -40,6 +41,6 @@ void BraveVPNMenuModel::ExecuteCommand(int command_id, int event_flags) {
 }
 
 bool BraveVPNMenuModel::IsBraveVPNButtonVisible() const {
-  auto* prefs = browser_->profile()->GetPrefs();
-  return prefs->GetBoolean(brave_vpn::prefs::kBraveVPNShowButton);
+  return g_browser_process->local_state()->GetBoolean(
```

Comment:
This one can stay as a profile preference, IMO. Basically, this is just whether or not the button shows next to the hamburger

Reply:
+1

Reply:
done